core/eds_test.go

```diff
@@ -28,16 +27,19 @@ func TestTrulyEmptySquare(t *testing.T) {
 }
 
 // TestNonEmptySquareWithZeroTxs tests that a non-empty square with no
-// transactions or blobs computes the correct data root (not the minimum DAH).
+// transactions or blobs computes the correct data root the minimum DAH.
```

Comment:
How can a square be non-empty if it contains no transactions or blobs? Is this referring to `types.Data` being non-empty because `SquareSize: 16`? If so, proposal to use "data" instead of "square" here because the latter makes me think of the ODS.

Would something like this work? "TestNonZeroSquareSize tests that the DAH hash of a block with no transactions == the DAH hash for an empty root even if SquareSize is set to something non-zero. Technically, this block data..."

Reply:
much better thanks

https://github.com/celestiaorg/celestia-node/pull/2253/commits/16780997ed610c7e70fce315c7da91a7d923e1c0